guides/error_handling.md

```diff
@@ -269,7 +269,7 @@ anymore! What if we create invalid users?
   ] |> Validated_list.sequence
 - : user list Validated_list.t =
 Preface_stdlib__.Validation.Invalid
- [Invalid_age (-32); Invalid_email "zaviervdwgmail.com"]
+ [Invalid_email "zaviervdwgmail.com"; Invalid_age (-32)]
```

Comment:
Why does the order change?

Reply:
I don't know why (I guess it is because of the tail-rec nature of the reimplementation) but it seems non problematic. We should try on YOCaml in order to watch if everything is ok.

Reply:
Rebuilding blogger using a compiled version of YOCaml based on this branch does not affect the output. So it seems fine.
Just sayin *as a commitolog* (lol), for the next patches, I think that you should split different concerns in different PRs.

Reply:
(Since YOCaml, *the one and only* is not affected, I took the liberty to *resolve the conversation* :)